docs/how-to-install-kubectl-helm.md

```diff
@@ -0,0 +1,448 @@
+This page shows you how to install kubectl and helm on macOS, Linux (Ubuntu) and Windows
+
+___
+
+# Menu
+
+- Install `kubectl` for each OS you need:
+	+ [Mac Os](#kubect_mac_os)
+	+ [Linux(Ubuntu)](#kubectl_linux)
+	+ [Windows](#kubectl_windows)
+
+- Install `helm` for each OS you need:
+	+ [Mac Os](#helm_mac_os)
+	+ [Linux(Ubuntu)](#helm_linux)
+	+ [Windows](#helm_windows)
+
+# Kubectl
+``kubectl`` is the Kubernetes command line tool, which can be used to deploy settings to the cluster.
+In this section, you should know how to install kubectl.
+
+## <a name="kubect_mac_os"></a>Mac Os
+
+### Install
+
+- Latest version:
+
+	+ If you are using [Homebrew](https://brew.sh/) package manager, you can install latest `kubectl` version with Homebrew.
+
+	Run the installation command:
+
+	```bash
+	$ brew install kubernetes-cli
+	$ brew link --overwrite kubernetes-cli # to ensure override new kubectl version to path
+	```
+
+	+ Install latest version with `kubectl` binary using `curl`(please install `curl` if you don't have it).
+
+	Run the installation command:
+
+	```bash
+	$ curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+	$ chmod +x ./kubectl
+	$ sudo mv ./kubectl /usr/local/bin/kubectl
+	```
+
+- Specific version:
+
+	+ If you have ever installed the desired version with `homebrew`, you can downgrade to that version.
+
+	```bash
+	$ brew info kubernetes-cli
+	$ brew switch kubernetes-cli 1.12.0 # example downgrade to version 1.12.0
+	```
+
+	+ Install a specific version:
+
+	For example, to download version v1.12.0 on macOS, type:

```

Comment:
In my previous doc, I also mentioned how to install a specific version with brew. https://www.benpickles.com/articles/72-downgrading-kubectl-with-homebrew